packages/docs/community/release-process/smart-contracts.md

```diff
@@ -91,6 +113,7 @@ Deploying a new contract release should occur with the following process:
       <ol>
         <li>Create a <code>release/contracts/N</code> branch at the desired commit.</li>
         <li>Submit this branch to a reputable third party auditor for review.</li>
+        <li>Begin drafting Release Notes and get team feedback.</li>
```

Comment:
Nit: It might be nice to make this less cLabs focused (team) and instead more community focused (community)

Reply:
👍  if kind of dawned on me halfway through that this is actually extremely public 😅  Do we still want to document drafting these up, or should that be documented internally elsewhere??

Reply:
I think the more transparent and prescriptive, the better! I'd imagine we'll be continuously adding to it as well